cmd/constants/marketmaps/markets.go

```diff
@@ -9673,6 +9673,409 @@ var (
 				{
 					"name": "alpaca_ws",
 					"off_chain_ticker": "AAPL"
+				},
+				{
+					"name": "polygon_api",
+					"off_chain_ticker": "AAPL"
+				}
+			]
+		},
+		"MSFT/USD": {
+			"ticker": {
+				"currency_pair": {
+					"Base": "MSFT",
+					"Quote": "USD"
+				},
+				"decimals": 8,
+				"min_provider_count": 1,
+				"enabled": true
+			},
+			"provider_configs": [
+				{
+					"name": "alpaca_ws",
+					"off_chain_ticker": "MSFT"
+				},
+				{
+					"name": "polygon_api",
+					"off_chain_ticker": "MSFT"
+				}
+			]
+		},
+		"GOOGL/USD": {
```

Comment:
hmm. interesting thing we need to bring up. how to handle diff class shares? GOOGL vs GOOG. same company, 2 different shares between of voting rights. I would assume we want to push the non voting rights one? or do we include both and average? question for Ondo

Reply:
Will ask this on Friday!